graf3d/eve7/src/REveDataCollection.cxx

```diff
@@ -79,28 +88,30 @@ void REveDataItemList::ItemChanged(Int_t idx)
 {
    std::vector<int> ids;
    ids.push_back(idx);
-   _handler_items_change(this, ids);
+   _handler_items_change( this , ids);
 }
 
 //______________________________________________________________________________
 
-void REveDataItemList::FillImpliedSelectedSet(Set_t &impSelSet)
+void REveDataItemList::FillImpliedSelectedSet( Set_t& impSelSet)
```

Comment:
According to clang-format it should be `Set_t &impSelSet` - please use it like this